scripts/setup_wizard/engine.py

```
@@ -0,0 +1,1632 @@
+        str(root / "scripts" / "rename-template.py"),
+        "--package",
+        identity.package_name,
+        "--name",
+        identity.project_name,
+        "--plugin-alias",
+        identity.plugin_alias or derived_plugin_alias(identity.package_name),
+        "--force",
+    ]
+    if identity.author:
+        command.extend(("--author", identity.author))
+    result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)